provider/testdata/data/baked/collator/meta_v1/es+es-u-co-trad+tr.rs.data

```diff
@@ -0,0 +1 @@
+::icu_collator::provider::CollationMetadataV1 { bits: 9u32 }
```

Comment:
Thought: I've never been super convinced about putting multiple locales in the variable name and now the file name. Seems like putting just one (the first alphabetically) is fine.